.github/workflows/ci_cd.yml

```diff
@@ -375,7 +373,7 @@ jobs:
           .venv\Scripts\Activate.ps1
           # Uninstall conflicting dependencies
           pip uninstall --yes vtk
-          pip install --extra-index-url https://wheels.vtk.org vtk-osmesa
+          pip install --extra-index-url https://wheels.vtk.org vtk-osmesa==9.2.20230527.dev0
```

Comment:
```suggestion
          pip install --extra-index-url https://wheels.vtk.org vtk-osmesa
```

Reply:
With the new release of vtk and pyvista, we are no longer bound to a specific version of vtk-osmesa and can use the latest release.